tools/worker/src/main/kotlin/io/bazel/Worker.kt

```diff
@@ -34,11 +35,17 @@ enum class Status(val exit: Int) {
 fun interface Worker {
     companion object {
         private const val PERSISTENT_WORKER = "--persistent_worker"
+        private val FLAGFILE_REGEX = Regex("""^--flagfile=((.*)-(\d+).params)$""")
```

Comment:
Don't really have a good feeling to be detecting flags via regex in this way. Not sure if there are any better way to do it though

Reply:
It was taken from rules_kotlin implementation https://github.com/bazelbuild/rules_kotlin/blob/5357ac1a1d25e6ce083c0cb7ee489012d396177d/src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/JdepsMerger.kt#L30 

I will investigate better ways separately